### ISSUE TITLE
Review: increasedLevel

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/upheno#334]
[increasedLevel.yaml](https://github.com/obophenotype/upheno/blob/master/src/patterns/increasedLevel.yaml)

could we rename this to increasedAmount?

**Comment by matentzn** [comment]
I find this pattern highly unspecific. I would like to know what it pertains to.. High amounts of what?

**Comment by nicolevasilevsky** [comment]
In HPO, it is used for increased levels of chemicals, cell counts, protein levels, biological processes (which doesn't really seem correct to me), and anatomical entities (which also seems odd)

**Comment by matentzn** [comment]
Yeah, that seems too wide a scope. Increased level is imho not the same as increased number of.. I think this pattern should apply to chemicals only.. @ybradford @sbello?

**Comment by sbello** [comment]
This came up in https://github.com/obophenotype/upheno/issues/327 as well. Yes, the desire is to split level of chemical from amount of physical entity (cells, tissues). We can either repurpose the non-specific level patterns or obsolete these and develop new ones.

**Comment by ybradford** [comment]
@matentzn Level to me implies chemical - or some measurable substance. I would not use it for increased number of cells, increased number of parts (anatomical),  - this should use increased number of and for process - this should use increased process quality or extra processual parts

**Comment by matentzn** [comment]
Okay, lets restrict the "level" patterns to chemicals and develop `has increased number of` patterns for the anatomical entities. Regarding increased "level" of process equivalents: rather than `extra processual parts`, which suggest to me that there is something qualitatively different going in the the process (something extra), perhaps `increased rate` is a better choice? @ybradford @sbello @nicolevasilevsky

**Comment by mah11** [comment]
>  Regarding increased "level" of process equivalents: rather than `extra processual parts`, which suggest to me that there is something qualitatively different going in the the process (something extra), perhaps `increased rate` is a better choice?

In FYPO, we have found it Very Useful Indeed to distinguish increased/decreased frequency or extent of occurrence of a process from increased/decreased "rate" in the sense of how fast it goes/how long it takes. We've been using the PATO 'occurrence' terms PATO:0002051/PATO:0002052 for the former and 'rate' PATO:0000911/PATO:0000912 for the latter. Happy to convert our design patterns if that's not accurate, but in any case we want to be able to continue to make the distinction between speed and occurrence.

I agree that neither 'increased rate' nor 'increased occurrence' looks like it means the same thing as 'extra processual parts'.

**Comment by matentzn** [comment]
@sbello @nicolevasilevsky @ybradford What do you think about @mah11 suggestion? should we have increasedRateOfProcess and increasedFrequencyOfProcess with the semantics Midori is suggesting?

**Comment by dosumis** [comment]
From discussion in meeting:

level vs amount in terrm names (do we even need to standardize?)
 - amount is favoured by ZP editors over levels in order to prevent churn, but we need (abnormal/increased/decreased levels as synonym).  ZP term names are automated so PATO term choice determines default term names here (unless over-ridden).
- 'levels' is currently used by MP and HP

Increased rate vs increased frequency:

PATO has rate as parent.  This seems reaosonable for many use cases, but we should discuss Midori's use cases at the workshop.

**Comment by matentzn** [comment]
@nicolevasilevsky can you tell me how you distinguish increased level from increased concentration terms in HPO?

**Comment by nicolevasilevsky** [comment]
I think increasedLevelOfChemicalEntity was an old name, which has now been replaced with increasedConcentrationOfChemicalEntity

**Comment by matentzn** [comment]
Make sure this is true :) because if I am not mistaken they are both there.

**Comment by nicolevasilevsky** [comment]
There is abnormallyIncreased and Decreased Level but not abnormal level

**Comment by nicolevasilevsky** [comment]
So I think for terms like this there is:
abnormalConcentrationOfChemicalEntity
abnormalConcentrationOfChemicalEntityInLocation

abnormallyDecreasedLevelOfChemicalEntity
abnormallyDecreasedLevelOfChemicalEntityInLocation

abnormallyIncreasedLevelOfChemicalEntity
abnormallyIncreasedLevelOfChemicalEntityInLocation

**Comment by nicolevasilevsky** [comment]
HPO currently uses 'amount' instead of 'concentration' for terms like HP_0045056 'Abnormal levels of alpha-fetoprotein'

**Comment by pnrobinson** [comment]
There are important differences between amount and concentration, and for instance in urine the concentration does not mean much, it is the amount in 24h. (Concentration in urine depends on factors like how much coffee you drink, but the amount in 24h does not). So please keep both. We may need to revise some of the definitions if this is not clear.

**Comment by nicolevasilevsky** [comment]
Thanks @pnrobinson!

@matentzn it sounds like we need both patterns, yes? I can create the additional patterns we need, please let me know if that sounds good

**Comment by matentzn** [comment]
@nicolevasilevsky Could you draft two detailed human-readable definitions that clarify exactly what the difference is between the two and update the two patterns accordingly; I dont quite understand it still, but I dont have too :P Maybe just add a comment here saying:

abnormalConcentrationOfChemicalEntity: "The amount of X per Y (or whatever)" 
abnormalLevelOfChemicalEntity: "..."

We will have the community review this afterwards.

**Comment by pfey03** [comment]
@nicolevasilevsky @matentzn 
Coming late into this, in Dicty we have level for chemical entities. For example have this phenotype
increased intracellular glucose level	
Def.: Increase in amount or accumulation of cytosolic glucose.
Despite the word 'amount' in the def, I would not use it here.
abnormalLevelOfChemicalEntity "the measured accumulation of a chemical" or such

**Comment by matentzn** [comment]
@nicolevasilevsky :P Sorry to be annoying; Just want to close this issue down soon; Thanks!

**Comment by nicolevasilevsky** [comment]
Not sure if I can properly define the difference between concentration and level, but I will try.

According to [wikipedia](https://en.wikipedia.org/wiki/Concentration), concentration is the abundance of a constituent divided by the total volume of a mixture. 

HPO only uses 'increased concentration' in one term:
**HP_0002910 'Elevated hepatic transaminase'**
'has part' some 
('increased concentration' and ('inheres in' some 
(protein and ('capable of' some 'transaminase activity') and ('produced by' some liver))) and ('has modifier' some abnormal))
**I don't think we have a pattern for this**

For abnormal/increased/decreased chemicals, the HPO is using the pattern abnormallyDecreasedLevelOfChemicalEntity (or increased)

For the purposes of HPO, I think we need these patterns:

abnormalLevelofChemicalEntity
abnormalLevelofChemicalEntityInLocation
(which would use the quality 'amount')
**we would need to add these**

abnormallyDecreasedLevelOfChemicalEntity
abnormallyDecreasedLevelOfChemicalEntityInLocation  

abnormallyIncreasedLevelOfChemicalEntity
abnormallyIncreasedLevelOfChemicalEntityInLocation
**these already exist**

**Comment by nicolevasilevsky** [comment]
Potential human readable defs are below:

abnormalLevelOfChemicalEntity
Deviation from normal level of a chemical entity or protein. 

abnormallyDecreasedLevelOfChemicalEntity
Reduction in the level of a chemical entity or protein.

abnormallyIncreasedLevelOfChemicalEntity
Elevation in the level of a chemical entity or protein.

**Comment by matentzn** [comment]
> abnormalLevelofChemicalEntity, abnormalLevelofChemicalEntityInLocation
   (which would use the quality 'amount') we would need to add these

(these are in the dosdp-patterns folder)

**Comment by nicolevasilevsky** [comment]
Ah, thanks @matentzn!

**Comment by LCCarmody** [comment]
So...looks like this ticket was never quite resolved? We have the same issue [here](https://github.com/obophenotype/human-phenotype-ontology/issues/7390).  Basically, we need an abnormalLevelofChemicalEntity and/or abnormallyIncreasedConcentrationofChemicalEntity/abnormallyDecreasedConcentrationofChemicalEntity.

I proposed in the HP ticket that we use abnormalLevelofChemicalEntity for Abnormal circulating cholesterol concentration to fit with the children of the term and most of HP terms. Should I start to make this pattern?
